### DISCUSSION TITLE
NR error in gateway with NLB-IP mode in aws load balancer

### DISCUSSION THREAD

**Discussion by parshva-shah-dsw** [istio/istio#53112] [General]
We have configured a nlb-ip mode load balancer for istio-gateway. by following the below documentation :-
https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.1/guide/service/nlb_ip_mode/
https://www.eksworkshop.com/docs/fundamentals/exposing/loadbalancer/ip-mode/#:~:text=The%20AWS%20Load%20Balancer%20Controller%20also%20supports%20creating,through%20the%20worker%20nodes%20in%20the%20Kubernetes%20cluster.

after making this changes in the load balancer mode we are getting NR error in the gateway while executing the requests the same gateway and virtual service configuration is being used which works in instance mode load balancer.

can we have any help here with any documentation on configuration with gateway

istioctl version 1.21.0
control plane version 1.21.0
data plane version 1.21.0 (95 proxies)

kubernetes 1.29.5 EKS

**Comment by zirain** [comment]
duplicated with https://github.com/istio/istio/issues/53113#issuecomment-2347190694
